### PR TITLE
chore: Update readme and gitchangelog section role

### PR DESCRIPTION
**Pull request by KazuCocoa**
- Addressed _1.0.0 will drop Python 2.0 support_ in README 
- Update changelog logic

---

I searched if the gitchangelog could insert an arbitrary line which was not in git log, but it seemed no. https://github.com/appium/python-client/pull/522/files#r415238307
So, let me leave a line about it. 
I also tweaked gitchangelog section role.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ An extension library for adding [Selenium 3.0 draft](https://dvcs.w3.org/hg/webd
 functionality to the Python language bindings, for use with the mobile testing
 framework [Appium](https://appium.io).
 
+# Notice
+
+**1.0.0 will drop Python 2.0 support**
```
Comment (mykola-mokhnach) [inline README.md:16]: Python 2.0 -> Python 2

Comment (mykola-mokhnach) [reply README.md:16]: Since v1.0.0 only Python 3 is supported

File: .gitchangelog.rc [hunk 2]
```diff
@@ -83,15 +83,14 @@ ignore_regexps = [
 ##
 section_regexps = [
     ('New', [
-        r'^[nN]ew\s*:\s*((dev|use?r|pkg|test|doc)\s*:\s*)?([^\n]*)$',
-     ]),
-    ('Changes', [
-        r'^[cC]hg\s*:\s*((dev|use?r|pkg|test|doc)\s*:\s*)?([^\n]*)$',
+        r'^([nN]ew|[fF]eat|[aA]dd)\s*(:|)\s*([^\n]*)$',
```
Comment (ki4070ma) [inline .gitchangelog.rc:86]: 👀  I'll follow this format in my following PRs.

Comment (KazuCocoa) [reply .gitchangelog.rc:86]: If you follow semantic pull request, `feat` will be here 🆗

**Pull request merged**